Library/Homebrew/os/linux.rb

```diff
@@ -11,13 +11,14 @@ module Linux
     sig { returns(String) }
     def os_version
       if which("lsb_release")
-        description = Utils.popen_read("lsb_release -d")
-                           .chomp
-                           .sub("Description:\t", "")
-        codename = Utils.popen_read("lsb_release -c")
-                        .chomp
-                        .sub("Codename:\t", "")
-        "#{description} (#{codename})"
+        lsb_info = Utils.popen_read("lsb_release -a")
+        description = lsb_info[/^Description:\s*(.*)$/, 1]
+        codename = lsb_info[/^Codename:\s*(.*)$/, 1]
+        if codename == "n/a"
```

Comment:
```suggestion
        if codename.blank? || (codename == "n/a")
```

as it occurred to be the codename also isn't worth outputting if, for whatever reason, it's missing entirely.

Reply:
I'm not sure it can be blank... @sjackman, do you know if it can or can't be blank?

Reply:
btw, we could also do `(codename: #{codename})` and in that case we won't have to check its value.

Reply:
> I'm not sure it can be blank... @sjackman, do you know if it can or can't be blank?

Even if we don't think it can't: it doesn't seem like a bad idea or much code to add?